### ISSUE TITLE
TopologyTestDriver doesn't work with Confluent.Kafka 1.9.2

### ISSUE THREAD

**Issue by alexander-bobrov** [LGouellec/kafka-streams-dotnet#167]
Streamiz.Kafka.Net nuget version: 1.3.1 and 1.3.0
OS Windows 10 Enterprise
net6.0

Code snippet below used to work, but now it throws an exception on TopologyTestDriver creation:

```
System.TypeLoadException
  HResult=0x80131522
  Message=Method 'DeleteGroupsAsync' in type 'Streamiz.Kafka.Net.Mock.Sync.SyncAdminClient' from assembly 'Streamiz.Kafka.Net, Version=1.3.1.0, Culture=neutral, PublicKeyToken=75e24dc15437bbc5' does not have an implementation.
  Source=<Cannot evaluate the exception source>
  StackTrace:
<Cannot evaluate the exception stack trace>

  This exception was originally thrown at this call stack:
    [External Code]
    Skai.TerminalGateway.Adapter.Tests.Tests.Test1() in MappingTest.cs
```

```
        [Test]
        public void Test1()
        {
            var config = new StreamConfig<StringSerDes, StringSerDes>();
            config.ApplicationId = "test-test-driver-app";

            StreamBuilder builder = new StreamBuilder();

            builder.Stream<string, string>("test")
                    .Filter((k, v) => v.Contains("test"))
                    .To("test-output");

            var t = builder.Build();
            var driver = new TopologyTestDriver(t, config);
        }
```

**Comment by alexander-bobrov** [comment]
Found the issue:
Confluent.Kafka package has been updated to 1.9.2 and it breaks the driver as we have >= 1.9.0 version requirement for net6.0

[Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka/) (>= 1.9.0)

**Comment by LGouellec** [comment]
Fix done, available the next release 1.3.0
